src/client/client.go

```diff
@@ -455,11 +455,6 @@ func NewForTest() (*APIClient, error) {
 
 // NewOnUserMachine constructs a new APIClient using $HOME/.pachyderm/config
 // if it exists. This is intended to be used in the pachctl binary.
-//
-// TODO(msteffen) this logic is fairly linux/unix specific, and makes the
-// pachyderm client library incompatible with Windows. We may want to move this
-// (and similar) logic into src/server and have it call a NewFromOptions()
-// constructor.
 func NewOnUserMachine(prefix string, options ...Option) (*APIClient, error) {
```

Comment:
Refactor `NewOnUserMachine` so we can reuse the lgoic for `NewEnterpriseClientOnUserMachine`.